hw10/8.py

```diff
@@ -0,0 +1,30 @@
+from Shark import Shark
+
+
+class Fish(Shark):
```

Comment:
The logic is not correct. 
Animal -> Fish -> Shark

Because Fish is more abstract than Shark